docs/how-to/index.rst

```diff
@@ -12,5 +12,8 @@ How-to guides are recipes that take the user through steps in key subjects. They
 
    Contribute to Batavia's code <contribute-code>
    Contribute to Batavia's documentation <contribute-docs>
+   Tour of the Code <tour>
+   Follow the Goat: Batavia's Test Suite <contribute-tests>
```

Comment:
This is a cute in-joke (I lol'd :-) but if you don't know about the testing goat, this is going to be a weird title. As much as I love puns, in this case, I think it detracts from the purpose. (plus, it's *obey* the testing goat... :-P )